### ISSUE TITLE
Import matlab trial data as arrays in Python

### ISSUE THREAD

**Issue by MusaMahmood** [MusaMahmood/SSVEP-Deep-Learning#3]
All data saved in 1250 x 257 trials, with 256 channels and 1 label column.

**Comment by MusaMahmood** [comment]
Duplicate of #1 - dealt with in pull request #4

**Comment by MusaMahmood** [comment]
Actually I haven't submitted updates to main_classify_keras.py.
